src/main/java/dev/morling/onebrc/CalculateAverage_ebarlas.java

```diff
@@ -21,13 +21,15 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Paths;
 import java.nio.file.StandardOpenOption;
+import java.util.Arrays;
 import java.util.List;
 import java.util.TreeMap;
 
 public class CalculateAverage_ebarlas {
 
-    private static final int HASH_FACTOR = 278;
-    private static final int HASH_MOD = 3_487;
+    private static final int MAX_KEY_SIZE = 64;
+    private static final int HASH_FACTOR = 433;
+    private static final int HASH_TBL_SIZE = 16_383; // range of allowed hash values, inclusive
```

Comment:
As the table is not growing, this assumes there will be no more than 16k stations.

This is not acceptable (see https://github.com/gunnarmorling/1brc/discussions/18#discussioncomment-8004520 and the remainder of the thread).